code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -151,7 +151,10 @@ var/const/MAX_ACTIVE_TIME = 400
 			M.Paralyse(MAX_IMPREGNATION_TIME/6) //something like 25 ticks = 20 seconds with the default settings
 	else if(iscorgi(M))

```

Comment:
I'm also half tempted to say this just shouldn't be a thing.

Reply:
I agree with Citinited here, this should be done in a separate PR.